framework/source/class/qx/data/controller/Form.js

```diff
@@ -278,6 +278,11 @@ qx.Class.define("qx.data.controller.Form",
       if (this.getTarget() == null) {
         return;
       }
+      else {
+        // if form was validated with errors and model changes
+        // the errors should be cleared see #8977
+        this.getTarget().getValidationManager().reset();
+      }
```

Comment:
As far as I see the validation manager is always created for the qx.ui.form.Form in the construct method, thus should be ok not to test it